dangerfile.js

```diff
@@ -53,7 +53,8 @@ function generateMDTable(headers, body) {
 
 /**
  * Generates a user-readable string from a percentage change
- * @param {string[]} headers
+ * @param {number} change
+ * @param {boolean} includeEmoji
  */
 function addPercent(change, includeEmoji) {
   if (!isFinite(change)) {
```

Comment:
I just reviewed some code. And found code on line 65
```javascript
if (/^-|^0(?:\.0+)$/.test(formatted)) {
```

I am not sure the purpose of this code, but when formatted equals `-.2abc` this `.test(formatted)` will also return true, does it meet the expectation ? Or is it a bug?

Reply:
`formatted` should never be an arbitrary string, so cases like `-2.abc` aren't really a concern.